README.md

```diff
@@ -129,7 +129,8 @@ the default Doctor settings. The default file contents are:
       raise: false,
       reporter: Doctor.Reporters.Full,
       struct_type_spec_required: true,
-      umbrella: false
+      umbrella: false,
+      include_hidden_doc: false
```

Comment:
Thoughts on renaming the option to `fail_hidden_doc` instead? I think that makes it more clear what the option does.

Reply:
@akoutmos Updated nomenclature, this error seems odd, locally it doesn't hit.
```
~/C/doctor ❯❯❯ MIX_ENV=test mix test                                                                                                                         add-hidden-doc-config
............................................
Finished in 0.4 seconds (0.1s async, 0.2s sync)
44 tests, 0 failures
```

Reply:
I reverted the reorganization so let's see where it fails now even though it fails for me locally now.